x-pack/plugins/search_playground/public/components/question_input.tsx

```diff
@@ -36,8 +36,10 @@ export const QuestionInput: React.FC<QuestionInputProps> = ({
     [onChange]
   );
   const handleKeyDown = useCallback((event: React.KeyboardEvent<HTMLTextAreaElement>) => {
-    if (event.key === keys.ENTER && !event.shiftKey) {
+    if (event.key === keys.ENTER && !event.shiftKey && !event.nativeEvent.isComposing) {
```

Comment:
can we have a small unit test, just so we can cover this use-case please?

Reply:
Hi @joemcelroy , Did you mean a unit test for this component or a specific scenario? It might be hard to simulate this scenario as we need to mock the form behavior and track key-down functionality. If this is the expectation, do we have any similar example I can follow?

Reply:
One idea for RTL cover this is:

```
  const textArea = screen.getByTestId('questionInput');

  fireEvent.compositionStart(textArea);

  fireEvent.keyDown(textArea, { key: 'Enter', shiftKey: false, nativeEvent: { isComposing: true } });

  // check to see if the submission method hasn't been called
  
  fireEvent.compositionEnd(textArea);
  fireEvent.keyDown(textArea, { key: 'Enter', shiftKey: false, nativeEvent: { isComposing: false } });

  // check to see if the submission method has been called
```